src/types/Fradrag.ts

```diff
@@ -3,7 +3,39 @@ import { Nullable } from '~lib/types';
 export interface Fradrag {
     type: Fradragstype;
     beløp: number;
-    beskrivelse: Nullable<string>;
+    utenlandskInntekt: Nullable<UtenlandskInntekt>;
+    delerAvPeriode: Nullable<DelerAvPeriode>;
+}
+
+export interface UtenlandskInntekt {
+    beløpIUtenlandskValuta: number;
+    valuta: string;
+    kurs: number;
+}
+
+export interface DelerAvPeriode {
+    fraOgMed: string;
+    tilOgMed: string;
+}
+
+export enum FradragObjectKeys {
+    type = 'type',
+    beløp = 'beløp',
+    fraUtland = 'fraUtland',
+    delerAvPeriodeChecked = 'delerAvPeriodeChecked',
+    utenlandskInntekt = 'utenlandskInntekt',
+    delerAvPeriode = 'delerAvPeriode',
+}
+
+export enum UtenlandskInntektKeys {
+    beløpIUtenlandskValuta = 'beløpIUtenlandskValuta',
+    valuta = 'valuta',
+    kurs = 'kurs',
+}
+
+export enum DelerAvPeriodeKeys {
+    fraOgMed = 'fraOgMed',
+    tilOgMed = 'tilOgMed',
 }
```

Comment:
Putt denne i `~lib/types`:
`export type KeyDict<T extends unknown> = { [key in keyof T]: key };`

Da kan vi gjøre dette:
```suggestion
export const FradragObjectKeys: KeyDict<Fradrag> {
    type: 'type',
    beløp: 'beløp',
    fraUtland: 'fraUtland',
    delerAvPeriodeChecked: 'delerAvPeriodeChecked',
    utenlandskInntekt: 'utenlandskInntekt',
    delerAvPeriode: 'delerAvPeriode',
}

export const UtenlandskInntektKeys: KeyDict<UtenlandsInntekt> = {
    beløpIUtenlandskValuta: 'beløpIUtenlandskValuta',
    valuta: 'valuta',
    kurs: 'kurs',
}

export const DelerAvPeriodeKeys: KeyDict<DelerAvPeriode> = {
    fraOgMed: 'fraOgMed',
    tilOgMed: 'tilOgMed',
}
```